src/inverseRobot.js

```diff
@@ -25,6 +25,16 @@
  */
 function inverseRobot(robot) {
   // write code here
+  const result = {};
+
+  for (const key in robot) {
+    if (result.hasOwnProperty(robot[key])) {
+      return null;
+    }
```

Comment:
Add a space before if statement)
```suggestion
    }
    
```

Reply:
I guess that Dima has meant that you should add a newline after!`if` clause to improve code readability